### DISCUSSION TITLE
[New Feature] TechnoType / SW Availability via Points

### DISCUSSION THREAD

**Discussion by ayylmaoRotE** [Phobos-developers/Phobos#886] [Ideas and feedback]
If possible to check how many points a player has, the Generals ranking system could be recreated e.g.:

```ini
[Superweapon]
SW.Points.Required=30000
```
This superweapon would be available after the player reaches this many points via kills. Combining this with UnitDelivery SWs + SW.NegBuildings tags, you could effectively create a choice between multiple items, like in Generals.

**Comment by Damfoos** [comment]
There should be a way to exclude units from contributing their exp to the overall amount, so stuff like ROTE hackers who get promoted as they hack without actually killing enemy units or veterancy applied by experience warhead do not get the player any unfair advantage. Perhaps a tag like

> [TechnoType]
> ExperienceGlobal.Contribute=no

per techotype would be enough.

Units that share gained exp with other units (opentopped transports promoted when their passengers gain exp, spawners promoted by spawns, mc units getting exp from the units they control) should also be taken into account as possible exceptions.
